docs/excel/excel-add-ins-pivottables.md

```diff
@@ -221,7 +222,142 @@ Excel.run(function (context) {
 });
 ```
 
-## Slicers
+## Filter a PivotTable
+
+The primary method for filtering PivotTable data is with PivotFilters. Slicers offer an alternate, less flexible filtering method. 
+
+[PivotFilters](/javascript/api/excel/excel.pivotfilters) filter data based on a PivotTable's four [hierarchy categories](#hierarchies) (filters, columns, rows, and values). There are four types of PivotFilters, allowing calendar date-based filtering, string parsing, number comparison, and filtering based on a custom input. 
+
+[Slicers](/javascript/api/excel/excel.slicer) can be applied to both PivotTables and regular Excel tables. When applied to a PivotTable, slicers function like a [PivotManualFilter](#pivotmanualfilter) and allow filtering based on a custom input. Unlike PivotFilters, slicers have an [Excel UI component](https://support.office.com/article/Use-slicers-to-filter-data-249f966b-a9d5-4b0f-b31a-12651785d29d). With the `Slicer` class, you create this UI component, manage filtering, and can control its visual appearance. 
+
+See [Filter with PivotFilters](#filter-with-pivotfilters) and [Filter with slicers](#filter-with-slicers) to learn more about filtering data with Excel JavaScript APIs.
+
+### Filter with PivotFilters
+
+[PivotFilters](/javascript/api/excel/excel.pivotfilters) allow you to filter PivotTable data based on the four [hierarchy categories](#hierarchies) (filters, columns, rows, and values). In the PivotTable object model, `PivotFilters` are applied to a [PivotField](/javascript/api/excel/excel.pivotfield), and each `PivotField` can have one or more assigned `PivotFilters`. To apply PivotFilters to a PivotField, the field's corresponding [PivotHierarchy](/javascript/api/excel/excel.pivothierarchy) must be assigned to a hierarchy category. 
+
+#### Types of PivotFilters
+
+| Filter type | Filter purpose | Excel JavaScript API reference |
+|:--- |:--- |:--- |
+| DateFilter | Calendar date-based filtering. | [PivotDateFilter](/javascript/api/excel/excel.pivotdatefilter) |
+| LabelFilter | Text comparison filtering. | [PivotLabelFilter](/javascript/api/excel/excel.pivotlabelfilter) |
+| ManualFilter | Custom input filtering. | [PivotManualFilter](/javascript/api/excel/excel.pivotmanualfilter) |
+| ValueFilter | Number comparison filtering. | [PivotValueFilter](/javascript/api/excel/excel.pivotvaluefilter) |
+
+#### Create a PivotFilter
+
+To filter PivotTable data with a Pivot*Filter (such as a PivotDateFilter), apply the filter to a [PivotField](/javascript/api/excel/excel.pivotfield). The following four code samples show how to use each of the four types of PivotFilters. 
```

Comment:
```suggestion
To filter PivotTable data with a `Pivot*Filter` (such as a PivotDateFilter), apply the filter to a [PivotField](/javascript/api/excel/excel.pivotfield). The following four code samples show how to use each of the four types of PivotFilters. 
```

Reply:
I think that in this location and the following comment on line 257, Pivot*Filter is referring to an individual filter as a concept and not referring to a specific class, so it shouldn't be wrapped in code ticks. I think this logic would apply to every instance of a class using a * to denote variations. Plus, since Pivot*Filter can't be directly copy/pasted into code as a functional element, I think it's confusing to wrap it in code ticks. Perhaps in this line `PivotDateFilter` should be wrapped in code ticks.

Reply:
(Italics only used because the lone _*_ was causing weird italicization of the entire paragraph)

Reply:
That's fair. My precedent was the [*OrNullObject copy](https://docs.microsoft.com/en-us/office/dev/add-ins/develop/application-specific-api-model?view=excel-js-preview#ornullobject-methods-and-properties) Maybe that shouldn't be in code ticks.

Reply:
Thanks for the example! Yes, I also argue *OrNullObject shouldn't be wrapped in code ticks because it's a concept and not a literal element one can copy into code.

Reply:
Cool. Let's flag that as something to figure out outside of this PR. Maybe a topic for a future team meeting?

Reply:
Will do! Internal tracking info: [work item 4697626](https://office.visualstudio.com/OC/_workitems/edit/4697626/)